nimbus-features/fakespotFeature.yaml

```diff
@@ -1,6 +1,6 @@
 # The configuration for the Fakespot feature
 features:
-  fakespot-feature:
+  shopping2023:
     description: >
       The configuration setting for the status of the Fakespot feature
```

Comment:
We should probably adjust the descriptions as well and remove Fakespot here too.

Reply:
I kept traces of fakespot mentions only because `shopping2023` we might forget in time what it refers to. What do you think it should say?